src/App.js

```diff
@@ -1,41 +1,20 @@
 import styled from 'styled-components';
-import Dayentry from './components/Dayentry';
+import {Routes, Route} from 'react-router-dom';
+import FormPage from './pages/FormPage';
+import CalendarPage from './pages/CalendarPage';
 
 const StyledPage = styled.div`
   padding: 15px;
 `;
```

Comment:
Instead of creating a styled component only to add padding, I would suggest to add padding to your `body` element in your `GlobalStyles` :)